### ISSUE TITLE
Authentication Failure

### ISSUE THREAD

**Issue by t3chlady** [nickvane/Magento-RestApi#80]
I am using your API in multiple locations.  The authentication is working on one client but not on another.  I am getting an error message that I am missing a parameter, but the format of my requests are the same as far as I know.  I am using the same functions and calling sequences but one works and the other fails.  Are there any special settings on the Magento side that I should know about? I have been working on this for a long time now with no progress.

**Comment by rizwanqur** [comment]
have you solved your issue

**Comment by t3chlady** [comment]
Yes, we discovered something on the server side that was incorrectly configured.

Do you have an email address that I can use to reach you a little more quickly?

I promise not to abuse the privilege.

Thank you for your reply,

Sincerely,

Jean

[cid:image001.png@01D6507D.1889DAB0]
Jean Holland, President
111 Golfview Lane
Ann Arbor, MI 48103
734-761-9107   Office
734-732-1498   Cell

**Comment by rizwanqur** [comment]
Yes my email address is rizwan@familysoukventures.com

**Comment by t3chlady** [comment]
Thank you.  Noted.

[cid:image001.png@01D652E6.1B05A9C0]
Jean Holland, President
111 Golfview Lane
Ann Arbor, MI 48103
734-761-9107   Office
734-732-1498   Cell
